eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -69,6 +89,22 @@ jobs:
     workingDirectory: build
     displayName: ctest
 
+  # Make coverage targets (specified in coverage_targets.txt) and assemble
```

Comment:
Not a huge deal but it would might be worth trying to share these steps with https://github.com/Azure/azure-sdk-for-cpp/blob/master/eng/pipelines/templates/jobs/archetype-sdk-client.yml as I must say while reviewing this I thought this was already done and so I was a little confused :)